README.md

```diff
@@ -67,13 +67,17 @@ For a complete example see the [example workflow](.github/workflows/example.yml)
 
 ## Using private images
 
-If your deployment requires private image, you have to `docker login` in a step before running this action.
+If your deployment requires private image, use [**podman-login**](https://github.com/redhat-actions/podman-login) action in a step before running this action to authenticate to the container image registry.
```

Comment:
If your deployment requires a private image, run `podman-login` in a step before this action so you can pull the image.